packages/client/modules/userDashboard/components/GcalModal/GcalModal.tsx

```diff
@@ -181,6 +184,10 @@ const GcalModal = (props: Props) => {
     setInviteAll((inviteAll) => !inviteAll)
   }
 
+  const handleChangeVideoType = (option: 'meet' | 'zoom' | null) => {
```

Comment:
+1 I've seen `'meet' | 'zoom'` more than a few times. Shall we refactor into an exported string union type?
Maybe something like

`import {GcalVideoTypeEnum} from '../../../../StartTeamPromptMutation.graphql'`